src/data/pokemon/evolution.h

```diff
@@ -525,13 +525,13 @@ const struct Evolution gEvolutionTable[NUM_SPECIES][EVOS_PER_MON] =
     [SPECIES_FLABEBE_ORANGE_FLOWER] = {{EVO_LEVEL, 19, SPECIES_FLOETTE_ORANGE_FLOWER}},
     [SPECIES_FLABEBE_BLUE_FLOWER]   = {{EVO_LEVEL, 19, SPECIES_FLOETTE_BLUE_FLOWER}},
     [SPECIES_FLABEBE_WHITE_FLOWER]  = {{EVO_LEVEL, 19, SPECIES_FLOETTE_WHITE_FLOWER}},
-    [SPECIES_FLOETTE_YELLOW_FLOWER] = {{EVO_ITEM, ITEM_NONE,/*ITEM_SHINY_STONE*/ SPECIES_FLORGES_YELLOW_FLOWER}},
-    [SPECIES_FLOETTE_ORANGE_FLOWER] = {{EVO_ITEM, ITEM_NONE,/*ITEM_SHINY_STONE*/ SPECIES_FLORGES_ORANGE_FLOWER}},
-    [SPECIES_FLOETTE_BLUE_FLOWER]   = {{EVO_ITEM, ITEM_NONE,/*ITEM_SHINY_STONE*/ SPECIES_FLORGES_BLUE_FLOWER}},
-    [SPECIES_FLOETTE_WHITE_FLOWER]  = {{EVO_ITEM, ITEM_NONE,/*ITEM_SHINY_STONE*/ SPECIES_FLORGES_WHITE_FLOWER}},
+    [SPECIES_FLOETTE_YELLOW_FLOWER] = {{EVO_ITEM, ITEM_SHINY_STONE, SPECIES_FLORGES_YELLOW_FLOWER}},
+    [SPECIES_FLOETTE_ORANGE_FLOWER] = {{EVO_ITEM, ITEM_SHINY_STONE, SPECIES_FLORGES_ORANGE_FLOWER}},
+    [SPECIES_FLOETTE_BLUE_FLOWER]   = {{EVO_ITEM, ITEM_SHINY_STONE, SPECIES_FLORGES_BLUE_FLOWER}},
+    [SPECIES_FLOETTE_WHITE_FLOWER]  = {{EVO_ITEM, ITEM_SHINY_STONE, SPECIES_FLORGES_WHITE_FLOWER}},
     [SPECIES_PUMPKABOO_SMALL]       = {{EVO_TRADE, 0, SPECIES_GOURGEIST_SMALL}},
     [SPECIES_PUMPKABOO_LARGE]       = {{EVO_TRADE, 0, SPECIES_GOURGEIST_LARGE}},
     [SPECIES_PUMPKABOO_SUPER]       = {{EVO_TRADE, 0, SPECIES_GOURGEIST_SUPER}},
     [SPECIES_ROCKRUFF_OWN_TEMPO]    = {{EVO_LEVEL_DUSK, 25, SPECIES_LYCANROC_DUSK}},
-    [SPECIES_SINISTEA_ANTIQUE]      = {{EVO_ITEM, ITEM_NONE,/*ITEM_CHIPPED_POT*/ SPECIES_POLTEAGEIST_ANTIQUE}},
+    [SPECIES_SINISTEA_ANTIQUE]      = {{EVO_ITEM, ITEM_CHIPPED_POT, SPECIES_POLTEAGEIST_ANTIQUE}},
```

Comment:
```suggestion
    [SPECIES_SINISTEA_ANTIQUE]      = {{EVO_ITEM, ITEM_NONE, SPECIES_POLTEAGEIST_ANTIQUE}},
```
Breaks if merged with Item Expansion since it's not defined in Item Expansion yet.